src/components/wizard-internet/wizard-internet.js

```diff
@@ -8,17 +8,128 @@ export default module('app.components.wizard-internet', [])
     },
     // TODO: handle vpn
     controller: class WizardContactCtrl {
+
       constructor($scope) {
         'ngInject';
 
         this.newInternet = {};
+        this.internetTunnelEnabled = false;
+        this.meshTunnelEnabled = false;
+        this.internetTunnelConf = [
```

Comment:
We could call this `tunnelFiles` because it's independent of the `tunnelType` if we decide to remove this (see below)